assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -102,6 +135,52 @@ export function useHelpCenter() {
     [store.actions]
   );
 
+  const getReadTips = useCallback(async () => {
+    try {
+      actions.fetchReadStart();
+      const { meta } = await getCurrentUser();
+
+      actions.fetchReadSuccess({
+        read: meta.web_stories_onboarding ?? [],
+      });
+    } catch {
+      actions.fetchReadError();
+    }
+  }, [actions, getCurrentUser]);
+
+  useEffect(() => {
+    getReadTips();
+  }, [getReadTips]);
```

Comment:
Combining this all into the effect will make it 1 less level of indirection to see when we actually read the tips:
```
  useEffect(() => {
    const hydrateReadTips = async () => {
      try {
        actions.hydrateReadTipsStart();
        const { meta } = await getCurrentUser();
        actions.hydrateReadTipsSuccess({
          readTips: meta.web_stories_onboarding ?? {},
        });
      } catch {
        actions.hydrateReadTipsError();
      }
    }
    await hydrateReadTips();
  }, [actions, getCurrentUser]);
```

Reply:
I'm a little more used to promises, but do we ever have to worry about a case where this component unmounts then tries to call `hydrateReadTipsSuccess` ?
```
useEffect(() => {
  let isMounted = true;
  getCurrentUser()
    .then(({ meta }) => isMounted && 
      actions.hydrateReadTips({
          readTips: meta.web_stories_onboarding ?? {},
       })
     ).catch(actions.hydrateReadTipsError);
  return () => (isMounted = false);
}, [actions, getCurrentUser])
```